patch_recurrence/dehaze.py

```diff
@@ -54,8 +53,7 @@ def main():
 
     # Scale array must be in decreasing order
     # scaled_imgs = steps.scale(img, [1, 0.75, 0.5, 0.375, 0.3, 0.25]) -> 6 scales
-    #scaled_imgs = steps.scale(img, [1, 300.0 / 384, 200.0 / 384, 150.0 / 384, 120.0 / 384, 100.0 / 384])
-    scaled_imgs = steps.scale(img, [1])
+    scaled_imgs = steps.scale(img, [1, 300.0 / 384, 200.0 / 384, 150.0 / 384, 120.0 / 384, 100.0 / 384])
```

Comment:
For future - we should think of a general solution for this. We could keep a list of pending stuff here - https://github.com/martiansideofthemoon/blind-dehazing/issues